spec/requests/session_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe SessionsController, type: :request do
+  describe '#new GET /sessions/new' do
+    describe 'success' do
+      before { get new_session_path }
+      it { expect(response).to have_http_status :success }
+    end
+
+    describe "redirect this if user hasn't login" do
+      before { get root_path }
+      it { expect(response).to redirect_to(new_session_path) }
+    end
+  end
+
+  describe '#create POST /sessions' do
+    let(:login_params) { { user: { email: user.email, password: user.password } } }
```

Comment:
__[rubocop]__
* Line is too long. [83/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)